packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -254,20 +256,38 @@ export default function MainFeedLayout({
     return isLaptop ? cloudinary.feed.bg.laptop : cloudinary.feed.bg.tablet;
   };
 
-  const FeedPageComponent = shouldUseFeedLayoutV1 ? FeedPageLayoutV1 : FeedPage;
+  const getPadding = () => {
+    if (isV1Search) {
+      return '!pt-2';
+    }
+
+    return shouldUseFeedLayoutV1 && '!pt-0';
```

Comment:
I don't think this was the only change between the FeedPageLayoutV1 and FeedPage styled components. Any reason to remove it?

Reply:
Yeah, this is not the only difference, but after checking, the new one is not accurate. After merging from main, this branch became broken as well.

Reply:
Seems to be broken on mobile.

https://github.com/dailydotdev/apps/assets/13744167/af00c3a6-e98d-416e-b01b-552b490ccbc4

Reply:
Found a fix to keep the `FeedPageLayoutV1`.

Reply:
I'd rather keep FeedPageLayoutV1, and fix what is broken from whatever has been merged after. As if we want to remove it, then we would need to go back to the ticket this was created, to make sure we're considering every detail.

Reply:
Curiosity, did u realised what was breaking it?

Reply:
Some margins somewhere, can't exactly remember on which place.